metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -805,7 +805,22 @@
                           :exclude-metadata true
                           :latest true})]
       ;; create variable association tombstones and queue the variable association delete events
-      (tombstone-associations context assoc-type search-params false :variable))))
+      (tombstone-associations context assoc-type search-params false :variable)))
+  (when (= :generic-association assoc-type)
+    (let [search-params1 (cutil/remove-nil-keys
```

Comment:
can a more description name be given for search-params1 and search-params2?

Reply:
It's very difficult without making the name super long... the content of the parameter is self explanatory. Also the description in tombstone-associations helps explain what the search param is for.

Reply:
I understand now.